package.json

```diff
@@ -1,125 +1,257 @@
+      {
+        "key": "j",
+        "command": "list.focusDown",
+        "when": "amVim.configuration.shouldUseVimStyleNavigationInListView && listFocus && !inputFocus"
+      },
+      {
+        "key": "k",
+        "command": "list.focusUp",
+        "when": "amVim.configuration.shouldUseVimStyleNavigationInListView && listFocus && !inputFocus"
+      },
+      {
+        "key": "o",
+        "command": "list.toggleExpand",
+        "when": "amVim.configuration.shouldUseVimStyleNavigationInListView && listFocus && !inputFocus"
+      },
+      {
+        "key": "l",
+        "command": "list.select",
+        "when": "amVim.configuration.shouldUseVimStyleNavigationInListView && listFocus && !inputFocus"
+      },
+      {
+        "key": "h",
+        "command": "list.collapse",
+        "when": "amVim.configuration.shouldUseVimStyleNavigationInListView && listFocus && !inputFocus"
+      },
+      {
+        "key": "g g",
+        "command": "list.focusFirst",
+        "when": "amVim.configuration.shouldUseVimStyleNavigationInListView && listFocus && !inputFocus"
+      },
+      {
+        "key": "shift+G",
+        "command": "list.focusLast",
+        "when": "amVim.configuration.shouldUseVimStyleNavigationInListView && listFocus && !inputFocus"
+      },
+      {
+        "key": "ctrl+d",
+        "command": "list.focusPageDown",
+        "when": "amVim.configuration.shouldUseVimStyleNavigationInListView && listFocus && !inputFocus"
+      },
+      {
+        "key": "ctrl+u",
+        "command": "list.focusPageUp",
+        "when": "amVim.configuration.shouldUseVimStyleNavigationInListView && listFocus && !inputFocus"
+      },
+      {
+        "key": "escape",
+        "command": "closeFindWidget",
+        "when": "editorFocus && findWidgetVisible"
+      },
+      {
+        "key": "escape",
+        "command": "closeBreakpointWidget",
+        "when": "breakpointWidgetVisible && editorFocus"
+      },
+      {
+        "key": "escape",
+        "command": "closeMarkersNavigation",
+        "when": "editorFocus && markersNavigationVisible"
+      },
+      {
+        "key": "escape",
+        "command": "closeReferenceSearch",
+        "when": "referenceSearchVisible && !config.editor.stablePeek"
+      },
+      {
+        "key": "escape",
+        "command": "cancelRenameInput",
+        "when": "editorFocus && renameInputVisible"
+      },
+      {
+        "key": "escape",
+        "command": "closeAccessibilityHelp",
+        "when": "accessibilityHelpWidgetVisible && editorFocus"
+      },
+      {
+        "key": "escape",
+        "command": "closeReplaceInFilesWidget",
+        "when": "replaceInputBoxFocus && searchViewletVisible"
+      },
+      {
+        "key": "escape",
+        "command": "workbench.action.closeMessages",
+        "when": "globalMessageVisible"
+      },
+      {
+        "key": "escape",
+        "command": "workbench.action.closeQuickOpen",
+        "when": "inQuickOpen"
+      }
+    ]
+  },
```

Comment:
The indention of this file changed a lot probably due to the different version of `npm` we are using (Mine is `5.6.0`).
Cloud you please keep the original indention style?

Reply:
And, `keybindings` block is much more readable when properties are joined into 1 line.